### ISSUE TITLE
Cannot bundle with webpack

### ISSUE THREAD

**Issue by Jack-Works** [ethersphere/bee-js#555]
> npm i  @ethersphere/bee-js webpack webpack-cli

> src/index.js
```js
import { Bee } from '@ethersphere/bee-js'
console.log(Bee)
```

And run `npx webpack`, it reports:

```
WARNING in ./src/index.js 2:12-15
export 'Bee' (imported as 'Bee') was not found in '@ethersphere/bee-js' (module has no exports)

ERROR in ./node_modules/.pnpm/@ethersphere+bee-js@3.3.0/node_modules/@ethersphere/bee-js/dist/mjs/index.js 1:0-28
Module not found: Error: Can't resolve './bee' in 'C:\Users\Jack\Workspace\bee\node_modules\.pnpm\@ethersphere+bee-js@3.3.0\node_modules\@ethersphere\bee-js\dist\mjs'
Did you mean 'bee.js'?
BREAKING CHANGE: The request './bee' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 @ ./src/index.js 1:0-41 2:12-15
```

It looks like this package ships an invalid ESM bundle. (Node ESM requires a fully specified extension name).

**Comment by AuHau** [comment]
Interesting! We are aware of several issues with the last release and there will be a patch release soon, but I was not aware of this problem. Thanks!

I assume there is no specific webpack configuration?

**Comment by Jack-Works** [comment]
> I assume there is no specific webpack configuration?

Yes. It failing with default webpack 5 config

**Comment by AuHau** [comment]
This should be fixed with the lastest `3.3.1` release. Please let us known if the problem persists for you.
